containers/.env.template

```diff
@@ -7,6 +7,7 @@
 SECTIGO_USERNAME=""
 SECTIGO_PASSWORD=""
 SECTIGO_PROFILE="CipherTrace EE"
+SECTIGO_TESTING=true
```

Comment:
All of us need to add this to our `containers/.env` file to ensure that when we're using docker compose we're not hitting Sectigo.